assets/prometheus-k8s/prometheus.yaml

```diff
@@ -93,6 +93,7 @@ spec:
   listenLocal: true
   nodeSelector:
     beta.kubernetes.io/os: linux
+  tolerations: []
```

Comment:
I think these are going to get reset by jsonnet generation

Reply:
Yeah, that's why travis was complaining ;)